next-types.d.ts

```diff
@@ -342,3 +342,18 @@ interface DbUserSearch {
   readonly login: string;
   readonly full_name: string;
 }
+
+interface DBList {
+  id: string;
+  user_id: number;
+  name: string;
+  is_public: boolean;
+  created_at: string;
+  updated_at: string;
+  deleted_at: string | null;
+}
+
+interface PagedData<T> {
```

Comment:
This is a new type I've added. We have lots of types for the same thing that only differ on the type of data in the array. We could replace existing types with this generic, but that's for another day.